src/coreclr/jit/lsraarm64.cpp

```diff
@@ -741,6 +741,16 @@ int LinearScan::BuildNode(GenTree* tree)
             BuildDefsWithKills(tree, 0, RBM_NONE, killMask);
             break;
 
+#ifdef SWIFT_SUPPORT
+        case GT_SWIFT_ERROR_RET:
+            // Plus one for error register
+            srcCount = BuildReturn(tree) + 1;
+            killMask = getKillSetForReturn();
+            BuildDefsWithKills(tree, 0, RBM_NONE, killMask);
+            BuildUse(tree->gtGetOp1(), RBM_NONE);
+            break;
```

Comment:
For correctness purposes I think we need to all uses simultaneously before the kills, so `BuildUse` should be after `BuildReturn`. Also setting its candidates to `RBM_SWIFT_ERROR`  would be appropriate I think, to be consistent with what happens inside `BuildReturn`.

Reply:
Also, I think you should build the use for the error first, to match the expected execution order of the node.